src/Frontend/Page/User.hs

```diff
@@ -356,7 +350,7 @@ createdIdeas userId ideasQuery = do
                     (mapM getIdeaStats
                      =<< filter visibleByCurrentUser
                          <$> findIdeasByUserId userId))
-        ds <- commonIdeaSpaceDelegations cUser user
+        ds <- commonIdeaSpaceDelegations cUser user >>= mapM delegationFull
```

Comment:
thanks.  i did consider optimizing that, but since it's just a `Map` operation and doesn't go to disk, i am strongly in favor of optimizing this after profiling.